src/plugins/intel_gpu/src/graph/impls/ocl/prior_box.cpp

```diff
@@ -0,0 +1,87 @@
+// Copyright (C) 2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <prior_box/prior_box_kernel_ref.h>
+#include <prior_box/prior_box_kernel_selector.h>
+#include <prior_box_inst.h>
+
+#include <impls/implementation_map.hpp>
+#include <vector>
+
+#include "intel_gpu/runtime/error_handler.hpp"
+#include "primitive_base.hpp"
+
+namespace cldnn {
+namespace ocl {
+
+struct prior_box_impl : typed_primitive_impl_ocl<prior_box> {
+    using parent = typed_primitive_impl_ocl<prior_box>;
+    using parent::parent;
+
+    std::unique_ptr<primitive_impl> clone() const override {
+        return make_unique<prior_box_impl>(*this);
+    }
+
+    static primitive_impl* create(const prior_box_node& arg, const kernel_impl_params& impl_param) {
+        auto params = get_default_params<kernel_selector::prior_box_params>(impl_param);
+        auto& kernel_selector = kernel_selector::prior_box_kernel_selector::Instance();
+        const auto& primitive = arg.get_primitive();
+
+        params.min_size = primitive->attributes.min_size;
```

Comment:
I would declare ```const auto& attrs = primitive->attributes``` and use ```attrs``` in lines below = just to be more laconic. 
Not insist though.